lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -21,7 +21,9 @@
 from django.test import RequestFactory
 from edxmako.shortcuts import render_to_response  # pylint: disable=import-error
```

Comment:
are you sure `# pylint: disable=import-error` is needed (I just removed a lot of them)